samples/highcharts/demo/heatmap-bubbles/demo.js

```diff
@@ -0,0 +1,106 @@
+// Create the chart
+Highcharts.chart('container', {
+
+    title: {
+        text: 'Riskmap',
+        style: {
+            fontSize: '1em'
+        }
+    },
+
+    xAxis: {
+        categories: ['Rare', 'Unlikely', 'Possible', 'Likely', 'Almost Certain'],
+        min: 0,
+        max: 4
+    },
+
+    yAxis: {
+        categories: ['Insignificant', 'Minor', 'Moderate', 'Major', 'Critical'],
+        title: null,
+        min: 0,
+        max: 4
+    },
+
+    chart: {
+        marginTop: 40,
```

Comment:
The `marginTop` setting makes the plot area touch the export icon in dark mode. Also, if a user wants to use this config with a longer title that wraps, that will not work well with an explicit top margin. Usually it is better to leave this out of the config and let the defaults work out the position.